Astro 8/8 release notes (#2663)
astronomer/docs: astro/install-gcp-hybrid.md

:::caution

> This document applies only to [Astro Hybrid](hybrid-overview.md). To see whether you're an Astro Hybrid user, click your Workspace name in the upper left corner of the Cloud UI, click **Organization Settings**, then go to **Settings** > **General**. Your Astro product type is listed under **Product Type**.

Reviewer: ```suggestion
This document applies only to [Astro Hybrid](hybrid-overview.md). To see whether you're an Astro Hybrid user, click your Workspace name in the upper left corner of the Cloud UI, click **Organization Settings**. From the **General** page, your Astro product type is listed under **Product Type**.
```
